src/env/index.ts

```diff
@@ -1,3 +1,12 @@
+/**
+ * This file is consumed by fabric.
+ * The `./node` and `./browser` files define the env variable that is used by this module.
+ * The `./node` module sets the env at import time.
+ * The `./browser` module is defined to be the default env and doesn't set the env at all.
+ * This is done in order to support SSR.
```

Comment:
instead of referring to SSR that is a specific case and nomenclature of React, let's generally say `isomorphic code` that is how someone defined it broadly. 
https://en.wikipedia.org/wiki/Isomorphic_JavaScript